packages/docs/content/foundation/theme-colors.mdx

```diff
@@ -118,33 +124,47 @@ These colors are used largely for background blocks and large content areas. Whe
 These colors are used largely to indicate the status of something.
 Be cautious using these colors to signify something that may contradict what most people would expect the colors to be used for.
 
-Status colors can also be used with these aliases where `green`, `gold`, and `red` are identical to `success`, `warn`, and `error`. Use these alternate naming options when statuses aren't approproiate or give enough detail. For example, in some situations, `--color-red-dark` might be more accurate and descriptive than `--color-error-darker`.
```

Comment:
Removed b/c we are going to be taking non-semantica colors out of the theme in future work and don't want to promote the use of `--color-red-dark` and simialr